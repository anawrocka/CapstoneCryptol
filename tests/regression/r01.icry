:l r01.cry
all ltTen [0 .. 9] where ltTen x = x < 10
all ltTen [0 .. 10] where ltTen x = x < 10
any ltTen [100,99 .. 10] where ltTen x = x < 10
any ltTen [100,99 .. 9] where ltTen x = x < 10
