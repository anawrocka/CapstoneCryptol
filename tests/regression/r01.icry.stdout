Loading module Cryptol
Loading module Cryptol
Loading module Main
[warning] at <interactive>:1:1--1:42:
  Defaulting type parameter 'bits'
             of finite enumeration
             at <interactive>:1:11--1:19
  to 4
True
[warning] at <interactive>:1:1--1:43:
  Defaulting type parameter 'bits'
             of finite enumeration
             at <interactive>:1:11--1:20
  to 4
False
[warning] at <interactive>:1:1--1:48:
  Defaulting type parameter 'bits'
             of finite enumeration
             at <interactive>:1:11--1:25
  to 7
False
[warning] at <interactive>:1:1--1:47:
  Defaulting type parameter 'bits'
             of finite enumeration
             at <interactive>:1:11--1:24
  to 7
True
